### DISCUSSION TITLE
'secured' status

### DISCUSSION THREAD

**Discussion by Danny-dK** [UtrechtUniversity/yoda#165] [Q&A] [answered]
When I submit a folder to the vault and is successfully secured, it gets the label 'secured'.

![image](https://user-images.githubusercontent.com/45395070/196382711-7a99645f-2d51-4279-b89a-af4291f8e2f8.png)

When I add a new file to that folder in the research environment, the label 'secured' remains visible when the current contents are not secured in the vault. Should that label not disappear or specify something in the likes of 'current version not secured' as that specific folder contents is not secured (only a previous version of that folder is secured)?

**Comment by Danny-dK** [comment]
Any input?

**Reply by lwesterhof** [reply, marked as answer]
This is a historical artifact. Originally a folder would still be locked after it was secured in the vault space as a data package. The user had to unlock the folder to start working on it again. This behaviour was later changed and a folder would be automatically unlocked after securing the data in the vault space.

Would it make sense to display the secured label as is, but remove it as soon as data in that folder is modified?

**Reply by Danny-dK** [reply]
Yes, that would indeed be my preference and more correct (at least in my view).

**Reply by Danny-dK** [reply]
Thanks for #358
